src/bipartite.jl

```diff
@@ -47,6 +47,9 @@ function LightGraphs.has_edge(b::BipartiteView, s, d)
     end
 end
 
+function LightGraphs.has_vertex(b::BipartiteView, v::Integer) 
+   return v >= 1 && v <= LightGraphs.nv(b) 
```

Comment:
```suggestion
   1 <= v <= LightGraphs.nv(b) 
```
and it probably be a one-liner without `function`